recipes/r-devemf/meta.yaml

```
@@ -0,0 +1,94 @@
+{% set version = '4.5' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-devemf
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/devEMF_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/devEMF/devEMF_{{ version }}.tar.gz
+  sha256: ead1cb3dbb41bf4caacc77335452c6575b1b5e620aac1205c3a80467a0a13c0e
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  run_exports: '{{ pin_compatible("r-devEMF", max_pin="x.x") }}'
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ compiler('c') }}              # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
+    - {{ compiler('cxx') }}            # [not win]
+    - {{ compiler('m2w64_cxx') }}      # [win]
+    - {{ posix }}filesystem        # [win]
+    - {{ posix }}sed               # [win]
+    - {{ posix }}grep              # [win]
+    - {{ posix }}autoconf
+    - {{ posix }}automake          # [not win]
+    - {{ posix }}automake-wrapper  # [win]
+    - {{ posix }}pkg-config
+    - {{ posix }}make
+    - {{ posix }}coreutils         # [win]
+    - {{ posix }}zip               # [win]
+    - {{ stdlib("c") }}
+
+  host:
+    - r-base
+    - zlib
+    - fontconfig
+    - freetype
+
+  run:
+    - r-base
+    - {{ native }}gcc-libs         # [win]
+    - zlib
+    - fontconfig
+    - freetype
+
+test:
+  commands:
+    - $R -e "library('devEMF')"           # [not win]
+    - "\"%R%\" -e \"library('devEMF')\""  # [win]
+
+about:
+  home: https://github.com/plfjohnson/devEMF
+  license: GPL-3.0-or-later
+  summary: Output graphics to EMF+/EMF.
+  license_family: GPL3
+  license_file:
+    - '{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-3'
+
+extra:
+  recipe-maintainers:
+    - Fabian-Boehm
+
+# The original CRAN metadata for this package was:
```

Please include Conda Forge R Team:

```suggestion
    - conda-forge/r
```